### PR TITLE
Casperf fixes

### PR DESCRIPTION
**Pull request by tiagolobocastro**
build(container/images): reduce container image size
    
    Fixup the spdk rpath in the pre-fixup allowing for the default
    fixup phase to shrink.
    Split io-engine derivation into 2 outputs, to avoid pulling in all binaries.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    feat: add casperf container image
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    fix(io-engine-client): use MY_POD_IP is present
    
    When running from k8s pod, we should not have to specify our own grpc endpoint.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    chore: update auto assign list
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    fix(casperf/lvs): purge lvs by wiping the first 8MiB
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    fix(null_bdev): parse size with 64bit
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    chore: allow specifying release version of libspdk
    
    Allows us to test performance within a dev shell.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

File: io-engine/src/bdev/lvs.rs
Comment (dsharma-dc) [inline io-engine/src/bdev/lvs.rs:1]: Could you share an example how the LvsMode is passed into by casperf tool?
 Is the Purge mode only with casperf?

**Review by dsharma-dc — APPROVED** [review]

**Review by hrudaya21 — APPROVED** [review]

File: io-engine/src/bdev/lvs.rs
Comment (tiagolobocastro) [reply io-engine/src/bdev/lvs.rs:1]: `sudo casperf "lvol:///lvol-1?size=220GiB&lvs=lvs:///pool-1?mode=purge&disk=aio:///dev/sda"`
Yes

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

File: n.sh
Comment (dsharma-dc) [inline n.sh:1]: Is this script `n.sh` merged by mistake? :)

File: io-engine/src/bdev/lvs.rs [hunk 1]
```diff
@@ -233,42 +233,60 @@ impl Lvs {
...
+            LvsMode::Purge => {
+                Self::wipe_super(args.clone()).await?;
+                crate::lvs::Lvs::create_or_import(args).await
+            }
         }
         .map_err(|error| BdevError::CreateBdevFailedStr {
             error: error.to_string(),
             name: self.name.to_owned(),
         })
     }
```
Comment (abhilashshetty04) [inline io-engine/src/bdev/lvs.rs:246]: @tiagolobocastro documentation?

File: n.sh
Comment (tiagolobocastro) [reply n.sh:1]: doh, must have git added by mistake

Comment (dsharma-dc) [reply n.sh:1]: you'll remove it in a separate PR I'd hope.

**Pull request merged**